orb.yaml

```diff
@@ -81,7 +81,7 @@ commands:
       - run:
           name: "Create NAIS Github deployment"
           command: |
-            deployment-cli deploy create --cluster=<< parameters.environment >> --repository=<< parameters.repo >> --appid=<< parameters.github-app-id >> \
+            deployment-cli deploy create --cluster=<< parameters.environment >> --repository=<< parameters.repo >> --appid=<< parameters.github-app-id >> --ref=$CIRCLE_SHA1 \
```

Comment:
Kunne vi heller hatt det som en parameter? Slik at vi har `default: master`, også kan man spesifisere noe annet ved behov?

Reply:
Slik at vi matcher standardoppsett for `deployment-cli`?

Reply:
1. Å gjøre det konfigurerbart er sikkert bare bra.
2. Da blir sprøsmålet hva som skal være default konfig. Høres bra ut at orb-en og deployment-cli har samme default, for det er nok det bukerne forventer, ref. https://en.wikipedia.org/wiki/Principle_of_least_astonishment
3. Jeg ser ikke hvorfor man noen gang skulle ønske å bruke master som verdi her, så jeg synes det ser ut som at default i deployment-cli er feil og at vi burde bruke commitID (sha) både der og her i orben. Ellers vet jeg jo ikke hvilen commit som blir bygget. Hvis jeg kjører samme bygg to ganger kan det blir bygd to ulike versjoner av koden. Opplys meg gjerne på fordelene med "master" som verdi i denne innsillingen.

Reply:
@TheUnnamedDude har sikkert bedre oversikt enn meg, men tipper at med master så bruker man siste commit-sha1 fra master. Altså ikke at det står master som id.

Reply:
? Er ikke det det samme? Mener at ref kan være et branch-navn i følge Github sitt API. Da er det nyeste commit på den branchen som brukes. Det er nettopp dette jeg ønsker å unngå.